elements/rh-navigation-primary/rh-navigation-primary-item.css

```diff
@@ -297,14 +297,14 @@
         }
 
         &[open] {
-          /* stylelint-disable-next-line no-descending-specificity */
-          summary {
-            background-color: var(--_background-color);
-
+          & summary {
+            /* stylelint-disable-next-line no-descending-specificity */
```

Comment:
a why comment would be appreciated

Reply:
Instead of a why, I refactored to address the underlying issue.